### PR TITLE
HPCC-16850 Document expireDays for file spray

### PR DESCRIPTION
**Pull request by JamesDeFabia**
## Type of change:
- [x] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [ ] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [ ] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browers
  - [ ] The component(s) render as expected

### CODE REVIEW

**Comment by JamesDeFabia** [comment]
@AttilaVamos Please review

File: docs/ECLStandardLibraryReference/SLR-Mods/SprayVariable.xml [hunk 1]
```diff
@@ -29,16 +31,19 @@
   <emphasis>allowoverwrite</emphasis> <emphasis role="bold">] , [</emphasis>
   <emphasis>replicate</emphasis> <emphasis role="bold">] , [</emphasis>
   <emphasis> compress </emphasis> <emphasis role="bold">] , [</emphasis>
-  <emphasis> sourceCsvEscape </emphasis> <emphasis role="bold">])</emphasis>
-  <emphasis></emphasis></para>
+  <emphasis> sourceCsvEscape </emphasis> <emphasis role="bold">]</emphasis>,
```
Comment (AttilaVamos) [inline docs/ECLStandardLibraryReference/SLR-Mods/SprayVariable.xml]: It seems something wrong. My 'HPCC-16630 Add expiry parameter to spray commands' merged into 7.0.0 therefore the expireDays doesn't available in 6.4.0. 
@richardkchapman @ghalliday I can see two solutions:
1. Merge https://github.com/hpcc-systems/HPCC-Platform/pull/9470 down to 6.4.0
2. Retarget this PR to 7.0.0

**Review by AttilaVamos — CHANGES_REQUESTED** [review]

File: docs/ECLStandardLibraryReference/SLR-Mods/SprayVariable.xml (hunk 1 above)
Comment (AttilaVamos) [inline docs/ECLStandardLibraryReference/SLR-Mods/SprayVariable.xml]: Independently my comment above there are some other parameters missing from the STD.File.SprayDelimited() and STD.File.fSprayDelimited() functions argument list: failIfNoSourceFile, recordStructurePresent, quotedTerminator.

They are exists in the table later.

**Comment by AttilaVamos** [comment]
@JamesDeFabia back to you.

**Comment by JamesDeFabia** [comment]
@AttilaVamos Edits made, back to you

File: docs/ECLStandardLibraryReference/SLR-Mods/SprayVariable.xml [hunk 2]
```diff
@@ -56,9 +66,15 @@
...
-  role="bold">]);</emphasis></para>
+  <emphasis> compress </emphasis> <emphasis role="bold">]</emphasis> ,
+  <emphasis role="bold"> [</emphasis> <emphasis> sourceCsvEscape </emphasis>
+  <emphasis role="bold">]</emphasis>, <emphasis role="bold"> [</emphasis>
+  <emphasis> failIfNoSourceFile </emphasis> <emphasis
+  role="bold">]</emphasis>, <emphasis role="bold"> [</emphasis> <emphasis>
+  recordStructurePresent </emphasis> <emphasis role="bold">]</emphasis>,
+  <emphasis role="bold"> [</emphasis> <emphasis> quotedTerminator </emphasis>
+  <emphasis role="bold">]</emphasis>, <emphasis role="bold">[</emphasis>
+  <emphasis>expireDays</emphasis> <emphasis role="bold">])</emphasis></para>
```
Comment (AttilaVamos) [inline docs/ECLStandardLibraryReference/SLR-Mods/SprayVariable.xml]: One parameter missing, the 'encoding' between 'quotedTerminator' and 'expireDays'.
'... [quotedTerminator ], [encoding ], [expireDays])'

File: docs/ECLStandardLibraryReference/SLR-Mods/SprayVariable.xml [hunk 3]
```diff
@@ -28,17 +30,25 @@
...
-  <emphasis></emphasis></para>
+  <emphasis> compress </emphasis><emphasis role="bold">]</emphasis> ,
+  <emphasis role="bold"> [</emphasis> <emphasis> sourceCsvEscape </emphasis>
+  <emphasis role="bold">]</emphasis>, <emphasis role="bold"> [</emphasis>
+  <emphasis> failIfNoSourceFile </emphasis> <emphasis
+  role="bold">]</emphasis>, <emphasis role="bold"> [</emphasis> <emphasis>
+  recordStructurePresent </emphasis> <emphasis role="bold">]</emphasis>,
+  <emphasis role="bold"> [</emphasis> <emphasis> quotedTerminator </emphasis>
+  <emphasis role="bold">]</emphasis>, <emphasis role="bold">[</emphasis>
+  <emphasis>expireDays</emphasis> <emphasis role="bold">])</emphasis></para>
```
Comment (AttilaVamos) [inline docs/ECLStandardLibraryReference/SLR-Mods/SprayVariable.xml]: One parameter missing, the 'encoding' between 'quotedTerminator' and 'expireDays'.
'... [quotedTerminator ], [encoding ], [expireDays])'

**Comment by AttilaVamos** [comment]
@JamesDeFabia one parameter missing from two places (see my comments).

**Comment by richardkchapman** [comment]
The Jira ticket says 6.4.0 but this PR targets master. Which is right?

**Comment by JamesDeFabia** [comment]
Should be 6.4.0, sorry. I changed it

**Comment by richardkchapman** [comment]
Can you squash before I merge. @AttilaVamos Are you happy now?

**Comment by AttilaVamos** [comment]
I can’t see the missing 'encoding' is added.

**Comment by JamesDeFabia** [comment]
I missed that, will add that now

**Comment by AttilaVamos** [comment]
@richardkchapman it is correct now.

**Comment by JamesDeFabia** [comment]
I corrected the missing element and squashed

**Pull request merged**